libpod/networking.go

```diff
@@ -54,13 +55,24 @@ func (r *Runtime) createNetNS(ctr *Container) (err error) {
 
 	resultStruct, err := cnitypes.GetResult(result)
 	if err != nil {
-		return errors.Wrapf(err, "error parsing result from CBI plugins")
+		return errors.Wrapf(err, "error parsing result from CNI plugins")
 	}
 
 	ctr.state.NetNS = ctrNS
 	ctr.state.IPs = resultStruct.IPs
 	ctr.state.Routes = resultStruct.Routes
 
+	// We need to temporarily use iptables to allow the container
+	// to resolve DNS until this issue is fixed upstream.
+	// https://github.com/containernetworking/plugins/pull/75
+	for _, ip := range resultStruct.IPs {
```

Comment:
Also, you might want to verify that resultStruct.IPs != nil.  Sometimes that can happen.

Reply:
`range` handles nil fine, so think we're set

Reply:
done